app/actions/webNavigationActions.ts

```diff
@@ -3,7 +3,13 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import * as types from '../constants/webNavigationTypes'
```

Comment:
A bit confusing that this is called types now but it's ok.  Maybe we should in the future call it constants and for the TS types call it types.